src/index.ts

```diff
@@ -118,4 +140,24 @@ module.exports = class ServerlessPlugin {
       }
     }
   }
+
+  private handleTags() {
```

Comment:
I'd probably name this function something like `addTags` or `addServiceAndEnvTags` just to make it a little more clear what it's doing at a glance